### PR TITLE
dapp: Ignore non-directories in dapp-remappings

### PR DESCRIPTION
**Pull request by transmissions11**
Files from MacOS finder like .DS_Store and other junk can end up in the libs folder and mess up remappings, this should fix

### CODE REVIEW

File: src/dapp/libexec/dapp/dapp-remappings [hunk 1]
```diff
@@ -17,6 +17,11 @@ function findRemappings(prefix) {
   ls(`${prefix}/${process.env.DAPP_LIB}`).forEach(name => {
     var lib = `${prefix}/${process.env.DAPP_LIB}`
     var src = `${lib}/${name}/${process.env.DAPP_SRC}`.replace(/^.\//, "")
+    
+    // If the file has an extension, ignore.
+    if (name.includes(".")) {
```
Comment (d-xo) [inline src/dapp/libexec/dapp/dapp-remappings:22]: should we maybe early return if it's not a directory instead?

Comment (transmissions11) [reply src/dapp/libexec/dapp/dapp-remappings:22]: yeah probably more reliable will fix

Comment (transmissions11) [reply src/dapp/libexec/dapp/dapp-remappings]: done!

File: src/dapp/CHANGELOG.md [hunk 2]
```diff
@@ -16,7 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- Dapp debug respects DAPP_LINK_TEST_LIBRARIES
+- Dapp debug respects `DAPP_LINK_TEST_LIBRARIES`
+- Dapp remappings ignores non-directories in `DAPP_LIB`
```
Comment (d-xo) [inline src/dapp/CHANGELOG.md:20]: Can you move this to an `## Unreleased` heading pls

Comment (transmissions11) [reply src/dapp/CHANGELOG.md:20]: oop

**Comment by d-xo** [comment]
:sparkling_heart:

**Pull request merged**